Pod/Classes/UILabel+BonMotUtilities.h

```diff
@@ -0,0 +1,20 @@
+//
+//  UILabel+BonMotUtilities.h
+//  Pods
+//
+//  Created by Nora Trapp on 3/2/16.
+//
+//
+
+@import UIKit;
+@protocol BONChainable;
+
+@interface UILabel (BonMotUtilities)
+
+/**
+ *  Assign a @p BONChainable object to apply to the label text. When a new value is assigned to @p text the chain attributes will be applied.
+ *  If a new value is assigned directly to @p attributedText the @p bonChainable property will be set to @p nil.
+ */
+@property (nonatomic, copy) id<BONChainable> bonChainable;
```

Comment:
#req for properties, please use the order `copy, nonatomic` to match the rest of the project. (If some place in the rest of the project doesn't match that, we should fix it. It should match the [Raizlabs Cocoa Style Gudie](https://github.com/Raizlabs/Raizlabs-Cocoa-Style))